src/app/WriteClient.h

```diff
@@ -0,0 +1,141 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines read client for a CHIP Interaction Data model
```

Comment:
comment seems wrong (says 'read client' where as this is 'write client')

Consider dropping these kind of  comments - they take up vertical space and do not help in understanding code (obviously writeclient implements the Write Client)

Reply:
removed, thanks